prow/config/config.go

```diff
@@ -362,6 +374,30 @@ func loadConfig(prowConfig, jobConfig string) (*Config, error) {
 	return nc, nil
 }
 
+// Loads multiple paths of secrets and add them in a map.
+func LoadSecrets(paths []string) (map[string][]byte, error) {
+	secretsMap := make(map[string][]byte)
+
+	for _, path := range paths {
+		stat, err := os.Stat(path)
+		if err != nil {
+			return nil, err
+		}
+
+		if stat.IsDir() {
```

Comment:
Why do we need a specific callout for this and the `os.Stat` call? Won't `ioutil.ReadFIle` fail in those cases and we can just bubble up that error?